src/features/Instructor/Profile/index.jsx

```diff
@@ -0,0 +1,84 @@
+import React, { useContext, useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { AppContext } from '@edx/frontend-platform/react';
+import {
+  Link,
+} from 'react-router-dom';
+
+import {
+  Tabs,
+  Tab,
+  Container,
+} from '@edx/paragon';
+import { ProfileCard, formatUTCDate } from 'react-paragon-topaz';
+
+import { fetchInstructorProfile } from 'features/Instructor/data';
+import { fetchAllClassesData } from 'features/Common/data';
+
+import { RequestStatus } from 'features/constants';
+
+const Profile = () => {
+  const dispatch = useDispatch();
+  const { authenticatedUser } = useContext(AppContext);
+  const instructorEmail = authenticatedUser.email;
+  const {
+    instructorImage,
+    instructorName,
+    lastAccess,
+    status,
+    created,
+  } = useSelector((state) => state.instructor.instructorInfo);
+  const instructorUserName = useSelector((state) => state.main.username);
+  const classes = useSelector((state) => state.common.allClasses.data);
+
+  const isLoading = status === RequestStatus.LOADING;
+
+  useEffect(() => {
+    if (instructorEmail) {
+      dispatch(fetchInstructorProfile(instructorEmail));
+      dispatch(fetchAllClassesData(instructorUserName));
+    }
```

Comment:
What would happen if I made changes using the institution selector?

Reply:
That is the reason I needed your changes

Reply:
a